web/yo/app/scripts/directives/curationQueue.js

```diff
@@ -277,98 +275,42 @@ angular.module('oncokbApp')
-                        if (article.link) {
-                            tempArr = tempArr.concat(['(', article.link, ')']);
-                        }
-                        if (article.variant) {
-                            tempArr = tempArr.concat(['Alteration:', article.variant + ',']);
-                        }
-                        if (article.tumorType) {
-                            tempArr = tempArr.concat(['Tumor type:', article.tumorType + ',']);
-                        }
-                        if (article.section) {
-                            tempArr = tempArr.concat(['Section:', article.section]);
-                        }
-                        content += tempArr.join(' ') + '\n';
-                    });
-                    content += '\nPlease try to curate this literature within two weeks (' + new Date(time + 12096e5).toDateString() + ') and remember to log your hours for curating this data.\n\n';
-                    content += 'If you have any questions or concerns please email or slack ' + adminName + '.\n\n';
+                    var content = 'Dear ' + item.get('curator').split(' ')[0] + ',\n\n';
+                    content += item.get('addedBy') + ' of OncoKB would like you curate the following publications in the indicated alteration, tumor type and section:\n\n';
+                    var tempArr = [item.get('article')];
+                    if (item.get('link')) {
+                        tempArr = tempArr.concat(['(', item.get('link'), ')']);
+                    }
+                    if (item.get('variant')) {
+                        tempArr = tempArr.concat(['Alteration:', item.get('variant') + ',']);
+                    }
+                    if (item.get('tumorType')) {
+                        tempArr = tempArr.concat(['Tumor type:', item.get('tumorType') + ',']);
+                    }
+                    if (item.get('section')) {
+                        tempArr = tempArr.concat(['Section:', item.get('section')]);
+                    }
+                    content += tempArr.join(' ') + '\n';
+                    content += '\nPlease try to curate this literature within two weeks (' + new Date(item.get('addedAt') + 12096e5).toDateString() + ') and remember to log your hours for curating this data.\n\n';
+                    content += 'If you have any questions or concerns please email or slack ' + item.get('addedBy') + '.\n\n';
                     content += 'Thank you, \nOncoKB Admin';
                     var subject = 'OncoKB Curation Assignment';
                     mainUtils.sendEmail(email, subject, content)
                         .then(function() {
-                            deferred.resolve();
+                            console.log('email sent');
                         }, function(error) {
-                            deferred.reject(error);
+                            console.log('error happened');
```

Comment:
If error happened, we need to notify the user that send email is not available, they need to manually notify them